sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+	ContainerPermissions.DELETE:ContainerPermissions
+	ContainerPermissions.LIST:ContainerPermissions
+	ContainerPermissions.READ:ContainerPermissions
+	ContainerPermissions.WRITE:ContainerPermissions
+	read:bool
+	write:bool
+	delete:bool
+	list:bool
+	_str:str
+
+	def __add__(self, other): ...
+	def __init__(self, read: bool=False, write: bool=False, delete: bool=False, list: bool=False, _str: str=None, *, ContainerPermissions.DELETE, ContainerPermissions.LIST, ContainerPermissions.READ, ContainerPermissions.WRITE): ...
+	def __or__(self, other): ...
+	def __str__(self): ...
+
+class ContainerProperties(DictMixin, object):
+	last_modified:datetime
+	etag:str
+	lease:~azure.storage.blob.models.LeaseProperties
+	public_access:str
+	has_immutability_policy:bool
+	has_legal_hold:bool
+	metadata:dict
+
+	def __init__(self, *, last_modified, etag, lease, public_access, has_immutability_policy, has_legal_hold, metadata, **kwargs): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class ContainerPropertiesPaged(AsyncPageIterator, AsyncIterator, AsyncIterable, Generic, object):
+	service_endpoint:str
+	prefix:str
+	marker:str
+	results_per_page:int
+	location_mode:str
+	current_page
+	command:callable
+	continuation_token:str
+
+	def __aiter__(self): ...
+	def __anext__(self): ...
+	def __class_getitem__(cls, params): ...
+	def __init__(self, command: callable, prefix: str=None, results_per_page: int=None, continuation_token: str=None, *, service_endpoint, marker, location_mode, current_page, get_next, extract_data): ...
+	def __init_subclass__(cls, *args, **kwargs): ...
+	def __new__(cls, *args, **kwargs): ...
+
+class ContentSettings(DictMixin, object):
+	content_type:str
+	content_encoding:str
+	content_language:str
+	content_disposition:str
+	cache_control:str
+	content_md5:str
+
+	def __init__(self, content_type: str=None, content_encoding: str=None, content_language: str=None, content_disposition: str=None, cache_control: str=None, content_md5: str=None, **kwargs): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class CopyProperties(DictMixin, object):
+	id:str
+	source:str
+	status:str
+	progress:str
+	completion_time:datetime
+	status_description:str
+	incremental_copy:bool
+	destination_snapshot:datetime
+
+	def __init__(self, *, id, source, status, progress, completion_time, status_description, incremental_copy, destination_snapshot, **kwargs): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class CorsRule(Model, object):
+	allowed_origins:list(str)
```

Comment:
can allowed_* `Union[str, list[str]]` instead of `list[str]`. It'll be more friendly to pass in one item.